src/link_assembly_files.py

```diff
@@ -23,20 +23,27 @@
 import testing
 
 
-def create_outname(outdir, infile):
+def create_outname(outdir, infile, extras):
   """Create the output file's name."""
   basename = os.path.basename(infile)
-  outname = basename + '.wast'
+  linker = os.path.splitext(os.path.basename(extras['linker']))[0]
+  if linker == 'lld':
+    outname = basename + '.wasm'
+  else:
+    outname = basename + '.wast'
   return os.path.join(outdir, outname)
 
 
 def link(infile, outfile, extras):
   """Create the command-line for a linker invocation."""
   linker = extras['linker']
   basename = os.path.splitext(os.path.basename(linker))[0]
+  install_root = os.path.dirname(os.path.dirname(linker))
+  symfile = os.path.join(install_root, 'lib', 'syms.def')
```

Comment:
Is there a more descriptive name we can use than "syms.def"? I'm not sure whether these are test-only symbols or something else.